webpack/ForemanTasks/Components/TasksDashboard/TasksDashboardHelper.js

```diff
@@ -1,11 +1,11 @@
-import URI from 'urijs';
 import { getURIQuery } from 'foremanReact/common/helpers';
 
 import {
   TASKS_DASHBOARD_AVAILABLE_TIMES,
   TASKS_DASHBOARD_QUERY_KEYS_TEXT,
   TASKS_DASHBOARD_QUERY_VALUES_TEXT,
 } from './TasksDashboardConstants';
+import { updateURlQuery } from '../TasksTable/TasksTableHelpers';
```

Comment:
Can you use the foreman helper for URI query?

Reply:
Nope, this is not in foreman

Reply:
What I meant is that in Foreman we have the [urlHelpers](https://github.com/theforeman/foreman/blob/9394ecb277798c488312f46866ab5d2a319506c6/webpack/assets/javascripts/react_app/common/urlHelpers.js) file,
and since this code is being repeated all over Foreman & plugins, you can use the helpers or contribute your own because this is a very trivial usage.

Reply:
I will contribute it to foreman and replace it in a separate PR